src/routes/Onboarding/utils.tsx

```diff
@@ -370,7 +369,7 @@ export function TutorialRequestToast(props: OnboardingUtilDeps) {
   return (
     <div
       data-testid="onboarding-toast"
-      className="flex items-center gap-6 min-w-80"
+      className="flex items-center gap-6 min-w-md"
```

Comment:
Niice thank you

Reply:
@franknoirot sorry I touched the onboarding haha just wanted things consistent and ALIGNED hahaha